tools/cldr-code/src/main/java/org/unicode/cldr/util/VoteResolver.java

```diff
@@ -287,8 +297,11 @@ public ImmutableSet<Integer> getVoteCountMenu(Organization org) {
          * vetter.votes needs to be defined before we can set admin.voteCountMenu.
          */
         static {
-            admin.voteCountMenu = ImmutableSet.of(vetter.votes, admin.votes); /* Not LOCKING_VOTES; see canVoteWithCount */
-            tc.voteCountMenu = ImmutableSet.of(vetter.votes, tc.votes, PERMANENT_VOTES);
+            admin.voteCountMenu = ImmutableSet.of(
+                street.votes, vetter.votes, vetter.tcorgvotes, tc.votes, admin.votes, PERMANENT_VOTES);
+            /* Not LOCKING_VOTES; see canVoteWithCount */
+            tc.voteCountMenu = ImmutableSet.of(
+                street.votes, vetter.votes, vetter.tcorgvotes, tc.votes, PERMANENT_VOTES);
```

Comment:
@macchiati  @stenshamn  can you confirm this set of votes for TC?
- 1 (street)
- 4 (vetter) 
- 6 (TC org vetter)
- 50 (TC)
- 1000 (PERMANENT_VOTES)

Reply:
Looks right to me.